### ISSUE TITLE
Feedback: 'EntityFramework integration'

### ISSUE THREAD

**Issue by SimonCropp** [Particular/docs.particular.net#3108]
Feedback for 'EntityFramework integration' https://docs.particular.net/samples/entity-framework/

Location in GitHub: https://github.com/Particular/docs.particular.net/blob/master/samples/entity-framework/sample.md

ping @Particular/nhibernate-persistence-maintainers

Should this be moved to use the sql persistence?

**Comment by SzymonPobiega** [comment]
Yeah. And probably it should be called "EF Unit of Work sample" because the EF integration (without the UoW support) is already covered in SQL persistence data access sample.

**Comment by SzymonPobiega** [comment]
Ooops. I've just noticed that there is no `OnSaveChanges` method in SQL persistence so I can't build the unit of work just yet.

**Comment by DavidBoike** [comment]
Does the merge of https://github.com/Particular/NServiceBus.Persistence.Sql/pull/156 unblock this?

**Comment by SzymonPobiega** [comment]
@DavidBoike Yeah. So the plan would be to delete the existing sample and add a new one based on SQL persistence (V3 only). Is that OK?

**Comment by DavidBoike** [comment]
:+1:

**Comment by weralabaj** [comment]
Addressed in https://github.com/Particular/docs.particular.net/pull/3465
